tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_hkv.h

```diff
@@ -552,8 +553,8 @@ class TableWrapper {
     } else {
       wfile.reset(new RandomKVFile<K, V, uint64_t>(
           fs, filepath, dim, buffer_size, append_to_file));
-      status = reinterpret_cast<RandomKVFile<K, V, uint64_t>*>(wfile.get())
-                   ->open(keyfile, valuefile, "wb");
+      status.Update(reinterpret_cast<RandomKVFile<K, V, uint64_t>*>(wfile.get())
```

Comment:
absl "status" feature override the "=". It wouldn't in-place update the origin status object but create a new one and assign. That would cause a compiler warning--the status was not used.